alpakaScreenshotUpload/action.yml

```diff
@@ -0,0 +1,51 @@
+name: Upload Screenshots to Alpaka
+
+inputs:
+  BUILD_ID_ARTIFACT:
+    type: string
+    required: true
+  SCREENSHOTS_DIR:
+    type: string
+    required: true
+  ALPAKA_UPLOAD_KEY:
+    required: true
+runs:
+  using: "composite"
+  steps:
+    - name: load build_id
+      uses: actions/download-artifact@v4
+      with:
+        name: ${{ inputs.BUILD_ID_ARTIFACT }}
+
+    - name: Display build_id
+      shell: bash
+      run: cat build_id.txt
+
+    - name: 'Upload screenshots'
+      id: uploadScreenshots
+      shell: bash
+      env:
+        ALPAKA_UPLOAD_URL: https://alpaka.ubique.ch/v1/screenshots/upload
+        BUILD_ID: $(cat build_id.txt)
+        UPLOAD_KEY: ${{ inputs.ALPAKA_UPLOAD_KEY }}
+        SCREENSHOTS_DIR: ${{ inputs.SCREENSHOTS_DIR }}
+      run: |
+        curl_cmd="curl -X POST $ALPAKA_UPLOAD_URL"
+        for file in "$SCREENSHOTS_DIR"/*; do
+          [ -f "$file" ] && curl_cmd="$curl_cmd -F screenshots=@$file"
+        done
+        BUILD_ID=$(cat build_id.txt)
```

Comment:
Wieso wird hier erneut die Build ID aus dem File gelesen? Ist ja schon in der ENV drin